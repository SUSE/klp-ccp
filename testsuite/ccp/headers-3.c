#include "headers-3.h"

void pu_f(void)
{
	g();
}
