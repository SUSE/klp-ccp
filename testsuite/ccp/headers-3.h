static void g(void)
{}

void h(struct A*);

struct A* i(struct A*)
{}
