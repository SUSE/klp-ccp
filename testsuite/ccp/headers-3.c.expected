/* klp-ccp: from testsuite/ccp/headers-3.h */
static void g(void)
{}

/* klp-ccp: from testsuite/ccp/headers-3.c */
void klpp_pu_f(void)
{
	g();
}
